types/setting.go

```diff
@@ -1385,6 +1388,16 @@ var (
 		},
 	}
 
+	SettingDefinitionV2DataEngineCPUMask = SettingDefinition{
```

Comment:
Can there be a conflict between this and `V2DataEngineGuaranteedInstanceManagerCPU`? For example, `V2DataEngineGuaranteedInstanceManagerCPU` requests 1 vCPU while the mask tries to use 2 or more specific CPU cores.
Then, the next question is, do we need to add a webhook validation for this?

Reply:
Yeah, I'm also thinking about the two settings.
`V2DataEngineGuaranteedInstanceManagerCPU` is better to be larger than `V2DataEngineCPUMask`, and we can validate it in the validating webhook. WDYT?